rust_dev_preview/examples/auto-scaling/src/bin/list-autoscaling-groups.rs

```diff
@@ -30,7 +30,10 @@ async fn list_groups(client: &Client) -> Result<(), Error> {
     let groups = resp.auto_scaling_groups();
 
     for group in groups {
-        println!("Name:  {}", group.auto_scaling_group_name(),);
+        println!(
```

Comment:
Some of these changes include line wrap changes (like this one). They might affect line numbers in related READMEs.